src/main/java/org/folio/dew/batch/bulkedit/jobs/processidentifiers/BulkEditItemIdentifiersJobConfig.java

```diff
@@ -46,7 +48,7 @@
   @Bean
   public Job bulkEditProcessItemIdentifiersJob(JobCompletionNotificationListener listener, Step bulkEditItemStep) {
     return jobBuilderFactory
-      .get(ExportType.BULK_EDIT_IDENTIFIERS.toString().concat("_ITEM"))
+      .get(ExportType.BULK_EDIT_IDENTIFIERS + "-" + ITEM.getValue())
```

Comment:
"-" can be string constant.

Reply:
Moreover, this can be extracted to the separate method.

Reply:
Thank you, fixed